README.md

```diff
@@ -166,6 +171,39 @@ In fact, it is so common that we need to use a "last hour" parameter in jobs tha
 default offset for all the well known (aliased) schedules to +1 period. E.g for an `@hourly`
 schedule, the default offset is `PT1H`, and for a `@daily` schedule the offset is `P1D`
 
+#### `schedule[].secret` **[secret]**
+Secret is used to mount keys stored in [Kubernetes Secrets] into the container.
```

Comment:
I think it should be OK to have Kubernetes here for now as we don't actually have other means to store secrets. Plus, this is what we claim in the beginning of the doc: `A data processing job scheduler for Kubernetes`. ;)